### PR TITLE
feat(i18n): route module

### PR DESCRIPTION
**Pull request by TikWind**
i18n route

### CODE REVIEW

File: src/pages/Route/List.tsx [hunk 1]
```diff
@@ -42,16 +44,16 @@ const Page: React.FC = () => {
     //   dataIndex: 'priority',
     // },
     {
-      title: '描述',
+      title: formatMessage({ id: 'route.list.description' }),
       dataIndex: 'description',
     },
     {
-      title: '更新时间',
+      title: formatMessage({ id: 'route.list.edit.time' }),
```
Comment (juzhiyuan) [inline src/pages/Route/List.tsx:51]: .updatedAt would be better

Comment (juzhiyuan) [reply src/pages/Route/List.tsx:51]: Also, we could use global.updatedAt instead of current key.

File: src/pages/Route/List.tsx [hunk 2]
```diff
@@ -60,22 +62,22 @@ const Page: React.FC = () => {
...
               remove(record.id!).then(() => {
-                notification.success({ message: '删除记录成功' });
+                notification.success({ message: formatMessage({ id: 'route.list.delete.success' }) });
                 /* eslint-disable no-unused-expressions */
                 ref.current?.reload();
               });
             }}
-            okText="确定"
-            cancelText="取消"
+            okText={formatMessage({ id: 'route.list.confirm' })}
```
Comment (juzhiyuan) [inline src/pages/Route/List.tsx:76]: Not a good key, we could use global.confirm.ok and global.confirm.cancel instead.

File: src/pages/Route/List.tsx [hunk 3]
```diff
@@ -93,7 +95,7 @@ const Page: React.FC = () => {
         request={(params) => fetchList(params, search)}
         toolBarRender={(action) => [
           <Input.Search
-            placeholder="请输入"
+            placeholder={formatMessage({ id: 'route.list,input' })}
```
Comment (juzhiyuan) [inline src/pages/Route/List.tsx:98]: Please note the extra comma

File: src/pages/Route/components/CreateStep4/CreateStep4.tsx [hunk 4]
```diff
@@ -17,15 +18,18 @@ const style = {
 };
 
 const CreateStep4: React.FC<Props> = ({ form1, form2, redirect, ...rest }) => {
+
+  const { formatMessage } = useIntl();
+
   return (
     <>
-      <h2>定义 API 请求</h2>
+      <h2>{formatMessage({ id: 'route.create.define.api.request' })}</h2>
```
Comment (juzhiyuan) [inline src/pages/Route/components/CreateStep4/CreateStep4.tsx:26]: Do not need the long key, we could use some words here, e.g route.create.defineAPIRequest

File: src/pages/Route/components/CreateStep4/CreateStep4.tsx [hunk 5]
```diff
@@ -17,15 +18,18 @@ const style = {
...
       <Step1 {...rest} form={form1} disabled />
       {!redirect && (
         <>
-          <h2 style={style}>定义 API 后端服务</h2>
+          <h2 style={style}>{formatMessage({ id: 'route.create.define.api.backend.server' })}</h2>
```
Comment (juzhiyuan) [inline src/pages/Route/components/CreateStep4/CreateStep4.tsx:30]: Same here

File: src/pages/Route/components/ResultView/ResultView.tsx [hunk 6]
```diff
@@ -1,21 +1,24 @@
...
+
 const ResultView: React.FC<Props> = () => (
   <Result
     status="success"
-    title="提交成功"
+    title={formatMessage({ id: 'route.result.submit.success' })}
     extra={[
       <Button type="primary" key="goto-list" onClick={() => history.replace('/routes/list')}>
-        返回路由列表
+        {formatMessage({ id: 'route.result.return.list' })}
```
Comment (juzhiyuan) [inline src/pages/Route/components/ResultView/ResultView.tsx:18]: global.back

File: src/pages/Route/components/ResultView/ResultView.tsx [hunk 7]
```diff
@@ -1,21 +1,24 @@
...
   onReset?(): void;
 };
 
+const { formatMessage } = useIntl();
+
 const ResultView: React.FC<Props> = () => (
   <Result
     status="success"
-    title="提交成功"
+    title={formatMessage({ id: 'route.result.submit.success' })}
```
Comment (juzhiyuan) [inline src/pages/Route/components/ResultView/ResultView.tsx:15]: global.submit.success

**Comment by juzhiyuan** [comment]
@LiteSun Please take a look at this PR.

File: src/pages/Route/components/ResultView/ResultView.tsx [hunk 8]
```diff
@@ -1,14 +1,12 @@
 import React from 'react';
 import { Result, Button } from 'antd';
 import { history } from 'umi';
-import { useIntl } from 'umi';
+import { formatMessage } from 'umi';
```
Comment (juzhiyuan) [inline src/pages/Route/components/ResultView/ResultView.tsx:4]: Nope

Comment (juzhiyuan) [reply src/pages/Route/components/ResultView/ResultView.tsx:4]: Use useIntl please

File: src/pages/Route/components/ResultView/ResultView.tsx [hunk 9]
```diff
@@ -1,13 +1,16 @@
...
 import { history } from 'umi';
-import { formatMessage } from 'umi';
+import { useIntl } from 'umi';
 
 type Props = {
   onReset?(): void;
 };
 
-const ResultView: React.FC<Props> = () => (
+const ResultView: React.FC<Props> = () => {
```
Comment (juzhiyuan) [inline src/pages/Route/components/ResultView/ResultView.tsx:10]:
```
const ResultView: React.FC<Props> = () => {
  const { formatMessage } = useIntl();
  return (
    <Result
     xx
    />
  )
}
```

**Comment by juzhiyuan** [comment]
Not sure why you did a force push 😂😂

**Comment by juzhiyuan** [comment]
Please have a look at those existing comments, once all PRs are merged, I will do a release.

**Comment by TikWind** [comment]
> Please have a look at those existing comments, once all PRs are merged, I will do a release.

I have test, maybe you can merge, and then I have one more pr about menu.

**Comment by juzhiyuan** [comment]
Any updates?

**Comment by TikWind** [comment]
> Any updates?

I am doing this, wait a moment.

**Pull request merged**